schema.sql

```diff
@@ -1,9 +1,10 @@
-/* Database schema to keep the structure of entire database. */
+/*create table day 1 */
 
-postgres=# CREATE TABLE animals(
-postgres(# id INT PRIMARY KEY NOT NULL,
-postgres(# name TEXT NOT NULL,
-postgres(# date_of_birth DATE,
-postgres(# escape_attempts INT NOT NULL,
-postgres(# neutered BOOL,
-postgres(# weight_kg DECIMAL);
+CREATE TABLE animals(
+  id INT NOT NULL,
```

Comment:
[ OPTIONAL ] I highly recommend to mark the id column as primary key to make it unique and not null 🙏🏻👍🏻✨✨